src/commands/cloud-run/flare.ts

```diff
@@ -56,7 +56,12 @@ const LOG_CONFIGS: LogConfig[] = [
 export class CloudRunFlareCommand extends Command {
   public static paths = [['cloud-run', 'flare']]
 
-  private isDryRun = Option.Boolean('-d,--dry,--dry-run', false)
+  public static usage = Command.Usage({
+    category: 'Serverless',
+    description: 'Gather Cloud Run service configuration and sends it to Datadog.',
+  })
+
+  private isDryRun = Option.Boolean('-d,--dry', false)
```

Comment:
It seems we are removing the previous PR that added `--dry-run`?

Reply:
Ouch, I messed up with merge conflicts, thanks a lot! 😅